.github/workflows/auto-publish.yml

```
@@ -0,0 +1,44 @@
+name: Publish GitHub release
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    name: Publish GitHub release
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Get version
+        id: version
+        run: |
+          VERSION=$(sed -n 's/.*APIVersion = "\(.*\)".*/\1/p' stytch/config/version.go)
+          echo "version=$VERSION" >> $GITHUB_OUTPUT
+          echo "release_tag=v$VERSION" >> $GITHUB_OUTPUT
+
+      - name: Get changed files
+        id: files
+        uses: jitterbit/get-changed-files@v1
+
+      - name: Check for config.go diff
+        id: diff
+        run: |
+          FOUND=0
+          for changed_file in ${{ steps.files.outputs.all }}; do
+            if [[ $changed_file == "stytch/config/version.go" ]]; then
```

My inner shellcheck says to quote this to avoid errors with spaces in filenames (which we should never have, but...)
```suggestion
            if [[ "$changed_file" == "stytch/config/version.go" ]]; then
```